src/classes/trip.js

```diff
@@ -0,0 +1,11 @@
+import time from '../scripts/time';
+
+class Trip {
+  constructor(tripInfo, destination) {
+    this.tripLocation = tripInfo.destinyID;
+    this.tripStatus = time.isBetween(tripInfo.status, tripInfo.date.split('/'), time.daysFromDate(tripInfo.date.split('/'), tripInfo.duration));
+    this.year = tripInfo.date.split('/')[0];
+    this.destination = destination;
+  }
+}
```

Comment:
Not sure what your plan is but be thinking ahead about where you want to put your calculate trip cost method. I think there are strong arguments either way for it to live on Trip class or on the Destination class with the way the API data is formatted